### PR TITLE
feat(@angular-devkit/architect): expose builder schema

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: packages/angular_devkit/architect/src/architect.ts [hunk 1]
```diff
@@ -254,14 +259,18 @@ export class Architect {
     return new builderClass(context);
   }
 
+  getSchema(schemaPath: Path): Observable<JsonObject> {
+    return this._host.read(schemaPath).pipe(
+      concatMap((buffer) => of(JSON.parse(virtualFs.fileBufferToString(buffer)))),
```
Comment (hansl) [inline packages/angular_devkit/architect/src/architect.ts]: `map`?

Comment (filipesilva) [reply packages/angular_devkit/architect/src/architect.ts]: Is there a benefit in using `map` over `concatMap` here?

**Comment by filipesilva** [comment]
Note: to get the schema json you can do:
```
const target = architect.getTarget(targetOptions);
architect.getBuilderDescription(target).pipe(
  concatMap(description => architect.getSchema(builderDescription.schema))
);
```

File: packages/angular_devkit/architect/src/architect.ts (hunk 1 above)
Comment (hansl) [reply packages/angular_devkit/architect/src/architect.ts]: Remove the `of()`?

**Comment by filipesilva** [comment]
Superseded by https://github.com/angular/devkit/pull/497.

**Pull request closed without merging**